Paybox/System/Base/Request.php

```diff
@@ -147,10 +148,23 @@ public function getForm($options = array())
         $options['csrf_protection'] = false;
 
         $parameters = $this->getParameters();
-        $builder = $this->factory->createNamedBuilder('', 'form', $parameters, $options);
-
-        foreach ($parameters as $key => $value) {
-            $builder->add($key, 'hidden');
+        // If symfony version is 3.* then we use the FQCN for form types
+        // Else we use the IDs.
+        if (substr(Kernel::VERSION, 0, 1) === "3") {
```

Comment:
Hi,
Even if there is a possibility that SF usage is not a full stack usage. 
If the kernel is version 3.* the form components will be at least 3.0 which is enouth.

And thank you for version_compare()
I did not know that function. 
I will make another PR if you like, to correct.